en/attributes.html

```diff
@@ -605,10 +607,15 @@ <h3>Paged attribute tensor disadvantages</h3>
 Changing the <em>paged</em> setting (e.g. removing the option) on a running system might cause hard out of memory situations as
 without paged the content nodes will attempt loading the attribute into memory without the option for page outs.
 </li>
+<li>
+Using <em>paged</em> in combination with <a href="approximate-nn-hnsw.html">HNSW indexing</a> is <em>strongly</em> discouraged. 
+<em>HNSW</em> indexing
+also searches and read tensors during indexing, causing random access during feeding. Once the system memory reaches 100% utilization, 
```

Comment:
Might want to state that this is *total* system memory. The plain system memory metric does not include what the OS uses for memory mapping / paging these attributes in/out of memory.